src/components/Content/Page.css

```diff
@@ -1,3 +1,8 @@
+.remote-heading {
+  margin: 30px 0 50px;
+  font-size: 40px;
```

Comment:
Same as above (except the centering).
I would also not call it .remote-heading, but instead just .heading because we don't include remote HTML content but only a string containing the title.

Reply:
done